content/00.build/40.tooling/20.hardhat/00.introduction/02.installation.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Installation
+description: How to install zksync-hardhat for your project.
+tags: ["zksync", "hardhart", "plugins", "installation"]
+---
+
+Before exploring ZKsync development with Hardhat, ensure your environment is properly set up. Follow these steps to install the necessary tools and
+meet the prerequisites for a smooth development experience.
+
+### Prerequisites
+
+**System requirements:**
+
+- Ensure your machine meets the [system requirements](%%zk_git_repo_era-compiler-solidity%%/tree/main#system-requirements) for running ZKsync and
+Hardhat. For Windows users, using Windows Subsystem for Linux (WSL 2) is highly recommended for better compatibility.
+
+**Node.js and package manager:**
+
+- Install [Node.js](https://nodejs.org) if it's not already installed. We recommend using a version supported by Hardhat.
+- Choose a package manager: [Yarn](https://classic.yarnpkg.com/en/docs/install) or [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+**Familiarity with Ethereum development:**
+
+- Basic knowledge of deploying smart contracts on Ethereum is beneficial. Refer to
+   the [Contract Deployment](/build/developer-reference/ethereum-differences/contract-deployment) documentation if you need a refresher.
+
+**Wallet and testnet funds:**
+
+- You’ll need a wallet with sufficient Sepolia ETH to deploy contracts. You can obtain Sepolia ETH
+   from [network faucets](https://faucets.chain.link/sepolia) and bridge funds to ZKsync Era using [bridges](https://zksync.io/explore#bridges).
+- Ensure you know how to [export your private key from MetaMask](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
```

Comment:
Remove this line. Not everyone uses Metamask.

Reply:
That was added by the developers...

Reply:
Since we're going through making changes to the documents, we want to make improvements following our documentation styleguide. I am okay with removing this line.